ethdb/kv_remote.go

```diff
@@ -725,3 +723,27 @@ func (back *RemoteBackend) NetVersion() (uint64, error) {
 
 	return res.Id, nil
 }
+
+func (back *RemoteBackend) Subscribe(onNewEvent func(*remote.SubscribeReply)) error {
+	fmt.Println("subscribe!")
+	subscription, err := back.remoteEthBackend.Subscribe(context.Background(), &remote.SubscribeRequest{})
+	if err != nil {
+		fmt.Println("subscribe! err=", err)
+		return err
+	}
+	fmt.Println("subscription, no error", subscription)
+	for {
+		event, err := subscription.Recv()
+		if err == io.EOF {
+			fmt.Println("EOF")
+			break
+		}
+		if err != nil {
+			fmt.Println("error reading event", err)
```

Comment:
Should we remove those?

Reply:
yeah, forgot to change it to a log entry